docs/BestPractices.md

```diff
@@ -204,3 +205,41 @@ FROM node:alpine as app
 ## Copy built node modules and binaries without including the toolchain
 COPY --from=builder node_modules .
 ```
+
+
+## Smaller images without npm/yarn
+
+If you want to achieve an even smaller image size than the `-alpine`, you can omit the npm/yarn like this:
+
+```Dockerfile
+FROM node:18-alpine3.16 AS builder
+WORKDIR /build-stage
+COPY package*.json ./
+RUN npm ci
+
+# Copy the the files you need
+COPY . ./
+
+RUN npm run build
+
+
+# Make sure the alpine version is the same as in the build stage
+FROM alpine:3.16
```

Comment:
Should probably use the same tag as the builder. Either match that one or swap the other

Reply:
@nschonni If you look closer, the main idea is to use node image at the build stage and vanilla alpine in the final stage.

But I have created a version for a alpine version instead of comment, so it is move obvious now from the code itself.